.github/actions/build_vgg_resnet_action/action.yml

```diff
@@ -0,0 +1,75 @@
+name: Setup and build networks; VGG16, Resnet50
+description: Build and run VV16 and Resnet50 with oneAPI base toolkit and oneAPI-Construction-Kit
+
+inputs:
+  workspace:
+    description: 'Main workflow workspace'
+    default: ${{ github.workspace }}
+
+runs:
+  # We don't want a new docker just a list of steps, so mark as composite
+  using: "composite"
+  steps:
+    - name: setup python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.8'
+
+    - name: Install Python libraries and system dependencies
+      shell: bash
+      run: |
+        pip install requests numpy h5py Pillow
```

Comment:
If the below commands use `sudo`, I assume this is not run as root. In that case, best to explicitly use `pip install --user`, it will otherwise give you a warning that it can't do what you want and fall back to `--user`.

Reply:
Heh, you changed the other `pip install` that I didn't notice, while leaving this one. I assume that was accidental.

Reply:
Right yeah I missed that. Updated !